docs/index.md

```diff
@@ -334,6 +207,11 @@
           </tr>
         </thead>
         <tbody class="divide-y divide-gray-100">
+          <tr>
+            <td class="py-3 px-4 text-gray-900 border-r border-gray-100">Diamond</td>
+            <td class="py-3 px-4 text-gray-700 border-r border-gray-100">8000</td>
+            <td class="py-3 px-4 text-gray-700">Gold + Prominent logo placement, 4 total reserved seats, keynote mention, premium SWAG package.</td>
```

Comment:
```suggestion
            <td class="py-3 px-4 text-gray-700">Gold + Prominent logo placement, 4 total reserved seats, keynote mention, premium SWAG package.</td>
```

Not sure about 4 seats. I would still do 2, but say priority seats or something. Otherwise Shopify and others will say "we won't pay 8000 because we're not sending 4 people".

Reply:
Yeah sounds good to me, also we want to have more diverse group of people and 4 is like 15% of the participants.